Automattic/wp-calypso
# Assembler: Replace activation page with confirmation page

Related to:

- https://github.com/Automattic/wp-calypso/issues/84996

## Proposed Changes

This PR removes the "redundant" Activation page and replaces it with Confirmation page, with different contents depending on the scenarios:

<details>
<summary>(Click to see 1st iteration)</summary>

|New site<br>(no changes)|Existing site<br>(from Creatio 2)|Existing site<br>(from different theme)|
|-|-|-|
|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/303b2d58-a937-43b4-9fc8-6db4223265cc">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/688a6055-8f10-471b-9f1d-79812058b0f1">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/83774839-3f5e-4d0c-988a-f72c72685e8f">|

</details>

<details>
<summary>(Click to see 2nd iteration)</summary>

|New site<br>(no changes)<br><br>OR<br><br>Existing site<br>(from Creatio 2)|Existing site<br>(from different theme)|
|-|-|
|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/303b2d58-a937-43b4-9fc8-6db4223265cc">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/6d6dc8cc-c23e-4c13-8806-1336f9f0a83e">|


</details>

<details>
<summary>(Click to see 3rd iteration)</summary>


|New site<br>(no changes, as-is in production)<br><br>OR<br><br>Existing site<br>(from Creatio 2)|Existing site<br>(from different theme)|
|-|-|
|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/303b2d58-a937-43b4-9fc8-6db4223265cc">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/a4ca6e8b-7703-4ddd-bfd2-87d8500e32db">|

</details>


<details>
<summary>(Click to see 4th iteration)</summary>


|New site<br>(no changes, as-is in production)<br><br>OR<br><br>Existing site<br>(from Creatio 2)|Existing site<br>(from different theme)|
|-|-|
|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/303b2d58-a937-43b4-9fc8-6db4223265cc">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/8e509c72-31bf-4b23-9684-58143acc0749">|

</details>

|New site<br>(no changes, as-is in production)<br><br>OR<br><br>Existing site<br>(from Creatio 2)|Existing site<br>(from different theme)|
|-|-|
|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/303b2d58-a937-43b4-9fc8-6db4223265cc">|<img width="347" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/6f565ac9-304e-4fb2-8288-03a467cd5f97">|



This is based on my own imagination. Needs @autumnfjeld @lucasmendes-design to review the copy (including the button) and approve this. Other ideas welcome. 😄 

## Testing Instructions

1. Go to `/start` and complete the Assembler flow.
   - Verify that the confirmation page is like the FIRST column above.
2. Go to Appearance -> Themes, click "Design your own" button, and complete the Assembler flow again.
   - Verify that the confirmation page is like the FIRST column above.
3. Change your theme to something else.
4. Go to Appearance -> Themes, click "Design your own" button, and complete the Assembler flow.
   - Verify that the confirmation page is like the SECOND column above.

## Pre-merge Checklist
